extensions/resteasy-reactive/rest-client-reactive/runtime/src/main/java/io/quarkus/rest/client/reactive/runtime/DefaultMicroprofileRestClientExceptionMapper.java

```diff
@@ -9,11 +9,6 @@
 public class DefaultMicroprofileRestClientExceptionMapper implements ResponseExceptionMapper {
 
     public Throwable toThrowable(Response response) {
-        try {
-            response.bufferEntity();
```

Comment:
This sentence in the `bufferEntity()` javadoc makes me think it might not be a good idea:

>      * In case the message entity is backed by an unconsumed entity input stream, all the bytes of the original entity input
>     * stream are read and stored in a local buffer. The original entity input stream is consumed and automatically closed
>     * as part of the operation and the method returns {@code true}.

The `automatically closed` part in particular. Couldn't it be a problem to not close the stream?

Reply:
We probably need some kind of load test to make sure nothing is being leaked

Reply:
> This sentence in the `bufferEntity()` javadoc makes me think it might not be a good idea:
> 
> > ```
> > * In case the message entity is backed by an unconsumed entity input stream, all the bytes of the original entity input
> > * stream are read and stored in a local buffer. The original entity input stream is consumed and automatically closed
> > * as part of the operation and the method returns {@code true}.
> > ```
> 
> The `automatically closed` part in particular. Couldn't it be a problem to not close the stream?

Perhaps, this was the original idea of using this method. Though, I don't think this should be done in the exception mapper or even it's done in another part of the code (because we should have seen the issue mentioned in the javadoc already).

Having said that, the risk is high, so let's close the pull request. 

> We probably need some kind of load test to make sure nothing is being leaked

I will put this on my todo list. Because I'm still curious about it :)